addons/project_issue/migrations/7.0.1.0/post-migration.py

```diff
@@ -19,7 +19,25 @@
 #
 ##############################################################################
 from openupgrade import openupgrade
-from openerp import pooler
+from openerp import pooler, SUPERUSER_ID
+
+
+def migrate_categories(cr, pool):
+    category_ids = pool['crm.case.categ'].search(
+        cr, SUPERUSER_ID, [('object_id.model', '=', 'project.issue')])
+    for category in pool['crm.case.categ'].browse(
+            cr, SUPERUSER_ID, category_ids):
+        new_category_id = pool['project.category'].create(
+            cr, SUPERUSER_ID, {
+                'name': category.name,
+            })
+        cr.execute('select id from project_issue where categ_id=%s',
```

Comment:
pay attention to the extra 's' in the write below. But things go wrong here anyways, because this write triggers recalculation of all function fields, and we don't want that. I'm currently busy rewriting this in sql